examples/benchmarks/tablebench/src/test/tsconfig.json

```diff
@@ -1,14 +1,9 @@
 {
-	"extends": "@fluidframework/build-common/ts-common-config.json",
+	"extends": "../../../../../common/build/build-common/tsconfig.test.node16.json",
 	"compilerOptions": {
-		"lib": ["ES2019", "ES2018.Promise", "DOM", "DOM.Iterable"],
-		"noImplicitAny": true,
-		"noUnusedLocals": false,
-		"noImplicitOverride": true,
 		"rootDir": "./",
-		"outDir": "../../dist/test",
-		"types": ["node", "mocha"],
-		"resolveJsonModule": true,
```

Comment:
Can the deltas be noted in the description?
Things I see:
`"resolveJsonModule": true,`  - removed; if viable cool
`"noImplicitAny": true,` - removed and should be restored
`"noImplicitOverride": true,` - removed and should be restored
lib changes - okay to update?

Reply:
@jason-ha - I'm the author of TableBench.  All of these settings were unintentionally copy/pasted from another example package.

The only one that I added was 'resolveJsonModule' which is no longer necessary (I used to import a JSON file for the benchmark, now I synthetically generate statistically similar test data.)

Reply:
Those two `noImplicit*` settings are ones that we *should* have everywhere. They are stricter, so, I would _vote_ for keeping them.

Reply:
And you did :)